spec/tasks/fetch_resraurants_from_zomato_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+require 'rake'
+
+describe 'fetch_restaurants_from_zomato' do
+  before(:all) do
+    Rake.application.rake_require("../../lib/tasks/fetch_restaurants_from_zomato")
+    Rake::Task.define_task(:environment)
+  end
+
+  before(:each) do
+    Review.delete_all
+    Restaurant.delete_all
+  end
+
+  describe "test rake task" do
```

Comment:
I would just change here to `context` and describing the state of the object you're currently holding.